### PR TITLE
fix: 🎨 use cache.teia.rocks

### PR DESCRIPTION
**Pull request by melMass**
Draft to identify potential issues/needs.

### CODE REVIEW

File: src/components/item-info/index.js [hunk 1]
```diff
@@ -22,13 +23,18 @@ export const ItemInfo = ({
...
 }) => {
   const { syncTaquito, collect, fulfillObjktcomAsk, curate, acc } =
     useContext(HicetnuncContext)
 
+  const { walletBlockList } = useSettings()
+
   const [showSignStatus, setShowSignStatus] = useState(false)
+  const [restricted, setRestricted] = useState(false)
+
+  useEffect(() => {
```
Comment (xat) [inline src/components/item-info/index.js]: imo `useMemo` is the more fitting hook in this case. `restricted` is basically a derivation of `walletBlockList` and `creator`.

File: src/pages/objkt-display/index.js [hunk 2]
```diff
@@ -19,10 +20,10 @@ import { Page, Container, Padding } from '@components/layout'
 import { renderMediaType } from '@components/media-types'
 import { ItemInfo } from '@components/item-info'
 import { Menu } from '@components/menu'
-import { Info, Collectors, Swap, Burn, History } from './tabs'
+import { Info, Collectors, Swap, Burn, History, Transfer } from './tabs'
 import styles from './styles.module.scss'
 import './style.css'
-import { Transfer } from '@components/collab/show/Transfer'
+import useSettings from 'hooks/use-settings'
```
Comment (xat) [inline src/pages/objkt-display/index.js]: `hooks/use-settings` => `@hooks/use-settings`

File: src/pages/objkt-display/index.js [hunk 3]
```diff
@@ -78,10 +81,14 @@ export const ObjktDisplay = () => {
...
-    objkt.ban = getWalletBlockList()
+
+    if (nsfwList.includes(objkt.id)) {
+      objkt.content_rating = METADATA_CONTENT_RATING_MATURE
+    }
+
     await context.setAccount()
 
-    if (objkt.ban.includes(objkt.creator.address)) {
+    if (walletBlockList.includes(objkt.creator.address)) {
```
Comment (xat) [inline src/pages/objkt-display/index.js]: wondering if it wouldn't be better to make `walletBlockList` a hashmap to make these lookups less expensive.

File: src/pages/tags/index.js [hunk 4]
```diff
@@ -8,7 +8,7 @@ import { renderMediaType } from '@components/media-types'
 import { Page, Container } from '@components/layout'
 import { PATH } from '@constants'
 import styles from './styles.module.scss'
-import { getWalletBlockList } from '../../constants'
+import useSettings from 'hooks/use-settings'
```
Comment (xat) [inline src/pages/tags/index.js]: `@hooks`

**Comment by Zir0h** [comment]
2 small issues:
- pdf files are no longer loading, I used to be able to flip the pages, now it hangs on "Loading PDF..." https://teia-ui-pr-187.onrender.com/objkt/774424
- video identicons don't load, it just says "identicon" https://teia-ui-pr-187.onrender.com/jjjjjohn

**Comment by xat** [comment]
> * video identicons don't load, it just says "identicon"

these issues might also be related to the refactorings.

File: src/pages/objkt-display/index.js (hunk 2 above)
Comment (melMass) [reply src/pages/objkt-display/index.js]: Yep, I have a few of them, that's vscode auto import, strangely it works 😅

File: src/pages/objkt-display/index.js (hunk 3 above)
Comment (melMass) [reply src/pages/objkt-display/index.js]: Like {address: restrictedStatus} ? I can look into that 👁️

**Comment by melMass** [comment]
> these issues might also be related to the refactorings.

Actually the logic for falling back to video was only used by media-types (image) component, for now I duplicated the logic for identicons. 🙈

File: src/pages/objkt-display/index.js (hunk 3 above)
Comment (xat) [reply src/pages/objkt-display/index.js]: jep, like that. this hashmap could be created in the `useSettings` hook.

**Pull request merged**